source/docs/transportation/alt-fuel-stations-v1/_record_fields.html.md.erb

```diff
@@ -1098,6 +1118,35 @@
       </td>
       <td class="doc-parameter-description">For electric stations, information about whether and how much users must pay to use the EVSE port, in French, if available.</td>
     </tr>
+    <tr>
+      <th class="doc-parameter-name" scope="row">restricted_access</th>
+      <td class="doc-parameter-value">
+        <div class="doc-parameter-value-field">
+          <strong>Type:</strong> boolean
+        </div>
+      </td>
+      <td class="doc-parameter-description">
+        <p>For public stations, an indication of whether the station has restricted access, given as a boolean:</p>
+        <table border="0" cellpadding="0" cellspacing="0" class="table table-bordered">
+          <thead>
+            <tr>
+              <th scope="col">Value</th>
+              <th scope="col">Description</th>
+            </tr>
+          </thead>
+          <tbody>
+            <tr>
+              <th scope="row">true</th>
+              <td>The station is open to any driver.</td>
+            </tr>
+            <tr>
+              <th scope="row">false</th>
+              <td>The station is reserved for patrons of a business, such as guests of a hotel, visitors of a museum, or customers of a retail store.</td>
```

Comment:
This might just be my own ignorance, but how does this differ from a private station? I guess I assumed things like these would maybe be labeled as private already? But I guess if private means something else, would it be worth clarifying what exactly we mean by private then, because we don't really have a definition of that and maybe other people would be confused like me?

Reply:
I don't know. Text is from ICF.